src/admin/BulkUserCreationPage.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC } from 'react';
+import { Box, Label, Heading, Container, Button, Text, Textarea, Alert, Select } from 'theme-ui';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+
+import { Role, CreateUserCommand } from '../api';
+import useBulkUserCreation from './hooks/useBulkUserCreation';
+
+const AnyBox = Box as any;
+
+const ROLES: Role[] = ['TRUSTED_PATIENT', 'CLINICIAN'];
+
+const BulkUserCreationPage: FC = () => {
+  const { create, loading, error, createdUsers } = useBulkUserCreation();
+
+  const form = useFormik<{ emailsString: string; role: Role }>({
+    initialValues: {
+      emailsString: '',
+      role: 'TRUSTED_PATIENT',
+    },
+    validationSchema: yup.object().shape({
+      emailsString: yup.string().required(),
+      role: yup.string().required(),
+    }),
+    onSubmit: ({ emailsString, role }) => {
+      const emails = getEmails(emailsString);
+      const command: CreateUserCommand[] = emails.map((email) => ({ email, roles: [role] }));
+      create(command);
+    },
+  });
+
+  return (
+    <Container variant="page">
```

Comment:
As this is a back-office page, it doesn't need to follow the mobile-specific layout, but I also saw no harm in it.